.github/workflows/build-ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: build-ci
+
+on:
+  push:
+    branches:
+      - 'main'
+
+  pull_request:
+    branches:
+      - 'main'
+
+  workflow_dispatch:
+
+env:
+  APP_NAME: yurii-web-app
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_AUTH_TOKEN }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+#      - name: Build and push
+#        uses: docker/build-push-action@v2
+#        with:
+#          file: Dockerfile
+#          push: true
+#          tags: ${{ secrets.DOCKER_HUB_USERNAME }}/${{ env.APP_NAME }}:latest
+#          cache-from: type=registry,ref=${{ secrets.DOCKER_HUB_USERNAME }}/${{ env.APP_NAME }}:buildcache
+#          cache-to: type=registry,ref=${{ secrets.DOCKER_HUB_USERNAME }}/${{ env.APP_NAME }}:buildcache,mode=max
```

Comment:
should we push docker images?

Reply:
was commented because dockerfile in another PR, uncommented